app/controllers/agent/ManageAgentController.scala

```diff
@@ -425,7 +425,8 @@ class ManageAgentController @Inject()(
           form = submitAgentAppointmentRequest,
           agentCode = agent.representativeCode,
           agentName = agent.name,
-          backLink = config.dashboardUrl("home")))
+          backLink = controllers.agent.routes.ManageAgentController.showManageAgent.url
+        ))
```

Comment:
This is halfway through a journey, so I don't understand why the back link was going home. I have updated it to go back to the previous page